# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../musl/pkg/ebtables/ebtables-ethernetdb.h-add-__THROW.patch
# Copyright (C) 2014 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From f9f9c612d457280ba3f9a6f5642c0fb820974cc5 Mon Sep 17 00:00:00 2001
From: Christian Wiese <chris@opensde.org>
Date: Mon, 28 Apr 2014 00:15:16 +0200
Subject: ethernetdb.h: add __THROW

---
 include/ethernetdb.h |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/include/ethernetdb.h b/include/ethernetdb.h
index 46d8bfd..faf7d0f 100644
--- a/include/ethernetdb.h
+++ b/include/ethernetdb.h
@@ -30,6 +30,10 @@
 #define	_PATH_ETHERTYPES	"/etc/ethertypes"
 #endif				/* _PATH_ETHERTYPES */
 
+#ifndef __THROW
+#define __THROW
+#endif
+
 struct ethertypeent {
 	char *e_name;		/* Official ethernet type name.  */
 	char **e_aliases;	/* Alias list.  */
-- 
1.7.2.3

